core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -260,11 +257,18 @@ func (l *logPollerWrapper) handleRouteUpdate(activeCoordinator common.Address, p
 		l.lggr.Debug("LogPollerWrapper: no changes to routes")
 		return
 	}
-	errActive := l.registerFilters(activeCoordinator)
-	errProposed := l.registerFilters(proposedCoordinator)
-	if errActive != nil || errProposed != nil {
-		l.lggr.Errorw("LogPollerWrapper: Failed to register filters", "errorActive", errActive, "errorProposed", errProposed)
-		return
+	zeroAddress := common.Address{}
+	if activeCoordinator != zeroAddress {
```

Comment:
you can put this inside registerFilters()

Reply:
mmm... yeah that's way cleaner. good suggestion